src/tests/ftest/erasurecode/offline_rebuild.yaml

```diff
@@ -1,27 +1,12 @@
 hosts:
   servers: !mux
     6_server:
-      test_servers:
-        - server-A
-        - server-B
-        - server-C
+      test_servers: server-[1-3]
     8_server:
-      test_servers:
-        - server-A
-        - server-B
-        - server-C
-        - server-D
+      test_servers: server-[1-4]
     12_server:
-      test_servers:
-        - server-A
-        - server-B
-        - server-C
-        - server-D
-        - server-E
-        - server-F
-  test_clients:
-    - client-A
-    - client-B
+      test_servers: server-[1-6]
```

Comment:
Same as above: We can't use `test_servers: 3` here?

Reply:
If we used:
```
    6_server:
      test_servers: 3
    8_server:
      test_servers: 4
    12_server:
      test_servers: 6
```
in CI - where only are providing 8 hosts with the `-ts` argument - then we would run out of hosts byt the time we got to `test_servers: 6`.